adhocteam/Head-Start-TTADP
# Add review page to navigator

**Description of change**
The review page is the last page of the navigator. It does not use the navigator form component like the other pages of the form. It only shows nav state when the form as been submitted.

**How to test**

1) Pull down changes
2) Go to the review section of the activity reports, the submit button is disabled
3) Completely fill out the form, clicking "continue" on every page
4) The submit button is no longer disabled
5) Click submit, note the state of the review page in the "side nav"
6) Go to the first page and unselect all grantees
7) The submit button is disabled

**Note**
This PR adds the mechanism for adding a review page. The current review page is a placeholder. There will be an additional PR with the review page as designed later. Let me know if you want me to keep splitting up PRs like this!

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/135

**Checklist**

- [n/a] Meets issue criteria
- [n/a] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
